### PR TITLE
(docz-core) handle props on windows

### PR DESCRIPTION
**Pull request by jesperorb**
### Description

Draft for fix for finding and parsing files on Windows (#1091 )

Not tested more than the supplied test that I am getting back props when setting state. Primarily [fastglob needs forward slashed](https://github.com/mrmlnc/fast-glob#how-to-write-patterns-on-windows) which it was not getting.

* Is there occasions where forward slashes should not be used in the state of the application, when is it a problem?

### CODE REVIEW

File: core/docz-core/__tests__/props.test.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      expect(cache).toBeNull()
+      const config = getTestConfig()
+      const pattern = getPattern(config)
+      const data = initial(config, pattern)
+      const params = mockedParams()
+      await data(params)
+      expect(readCacheFile()).not.toBeNull()
+    })
+  })
+})
```
Comment (rakannimer) [inline core/docz-core/__tests__/props.test.ts]: 👍

**Comment by rakannimer** [comment]
Nice PR ( and tests ) ! 👍 

> Is there occasions where forward slashes should not be used in the state of the application, when is it a problem?

Edge case for sure but maybe when escaping a character ?

For example : 

```sh
mkdir -p test/a@b
cd test/a\@b
```
